pkg/apis/controlplane/v1beta2/types.go

```diff
@@ -325,6 +325,16 @@ type NodeStatsSummary struct {
 	AntreaClusterNetworkPolicies []NetworkPolicyStats `json:"antreaClusterNetworkPolicies,omitempty" protobuf:"bytes,3,rep,name=antreaClusterNetworkPolicies"`
 	// The TrafficStats of Antrea NetworkPolicies collected from the Node.
 	AntreaNetworkPolicies []NetworkPolicyStats `json:"antreaNetworkPolicies,omitempty" protobuf:"bytes,4,rep,name=antreaNetworkPolicies"`
+	// List of PodMulticast statistics collected from the Node.
+	MulticastStats []MulticastStats `json:"multicastStats,omitempty" protobuf:"bytes,5,rep,name=multicastStats"`
+}
+
+// MulticastStats contains the multicast groups that a Pod has joined.
+type MulticastStats struct {
```

Comment:
It looks this struc is only for the group list per Pod? But the public API is shown pod list per group in MulticastGroupMember. Why use a different orgnization here?

Do you plan to show the multicast traffic stats in this API on the scenario no ANP is applied to the Pod?

Reply:
It looks this struc is only for the group list per Pod? But the public API is shown pod list per group in MulticastGroupMember. Why use a different orgnization here?
>  Updated. Please check https://github.com/antrea-io/antrea/pull/3354#discussion_r858831356

Do you plan to show the multicast traffic stats in this API on the scenario no ANP is applied to the Pod?
> No sure I understand your question. the group pods relation will be affected by multicast np. If igmp is block by np, the pod is not considered joined to the group in the final `antctl get multicaststats` representation. no ANP and ANP with allow action to the igmp are considered to be the same.

Reply:
I mean do you want to show the packet number in the MulticastStats as you only list the Pod count of a Group in this struct? Or the the multicast packet numbers are only shown in the result of ANP stats? But if a Multicast group is not applied with any ANP, we can't get the traffic numbers.

Reply:
If you want to check traffic statistics not covered by ANP, you can check the command `antctl get multicaststats` running in antrea-agent, as described here https://github.com/antrea-io/antrea/issues/3294.